src/Analysis.cpp

```diff
@@ -137,6 +136,61 @@ void buildCFG(CFG &topCFG, std::vector<std::string> branchOrder,
   }
 }
 
+std::vector<std::string> getAnnotationStrings(std::string optLoadFileName) {
+  std::string testName = getTestName(optLoadFileName);
+  std::string parentName = getParentName(optLoadFileName);
```

Comment:
Please document the format of `optLoadFileName`. I'm not sure what the purpose of this code is, since I would expect our tool to take a list of `.c` files as input.

(Related question: have you tested this on `.h` files? We should add at least one test with an annotation in a header file.)

Reply:
Turns out getTestName and getParentName are not necessary at all. I also kind of forgot why I had them in the first place. I think it was because I assumed `optLoadFileName` specified a `.ll` file, but after testing it out, it just specifies a `.c` file, so no extra parsing is needed. 

> have you tested this on .h files? 

If you mean where I directly feed in a `.h` file, no.  The command we run to test a .c file is just: 

```
command="make ; clang -emit-llvm -g -S -fno-discard-value-names -Xclang -disable-O0-optnone -c ../test/test$1.c -o ../test/test$1.ll ; opt -load CodeAnalyzer.so -CodeAnalyzer ../test/test$1.ll"
```
where `$1` is the number of the test (e.g., test1, test2, ...). Clang cannot generate IR for header files. I have tried feeding it a header file and it puked unparsable weird text encodings on my screen. I do have tests such as `test/test23/test23.c` and `test/test23/test24.c` that include header files with annotations from `test/test23/test23.h`, and those annotations do appear in the AST for the `.c` files.

Reply:
>  I do have tests such as test/test23/test23.c and test/test23/test24.c that include header files with annotations from test/test23/test23.h, and those annotations do appear in the AST for the .c files.

Thanks, that's definitely sufficient.